lib/sycamore/sycamore/query/planner.py

```diff
@@ -96,6 +105,7 @@ def __init__(
         self._os_client = os_client
         self._llm_client = llm_client or OpenAI(OpenAIModels.GPT_4O.value)
         self._use_examples = use_examples
+        self._natural_language_response = natural_language_response
```

Comment:
Can we just use the same flag for the client and executor? They’re opposites i think?

Reply:
They are the same but I made the command line switch for the CLI the opposite (`--raw-data`) since I think that is less likely to be desired in the case of running the CLI.